src/v/cluster/topic_table.cc

```diff
@@ -593,14 +593,47 @@ topic_table::apply(move_topic_replicas_cmd cmd, model::offset o) {
           new_replicas,
           tp->second,
           *assignment,
-          o);
+          o,
+          false);
     }
 
     notify_waiters();
 
     co_return errc::success;
 }
 
+ss::future<std::error_code>
+topic_table::apply(force_partition_reconfiguration_cmd cmd, model::offset o) {
```

Comment:
I think it would makes sense to verify if an union of current replica set and requested replica set is not empty. Otherwise force reconfiguration may lead into a situation in which replica set is reconfigured to a set of nodes that are completelly empty.